src/UnderlineNav2/UnderlineNav.Item.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import {Meta, Story} from '@storybook/react'
+import {BaseStyles, ThemeProvider} from '..'
+import {UnderlineNav} from './index'
+import {UnderlineNavItem} from './UnderlineNavItem'
+import {CodeIcon, GitPullRequestIcon, PeopleIcon} from '@primer/octicons-react'
+
+const icons = {CodeIcon, GitPullRequestIcon, PeopleIcon}
+
+export default {
+  title: 'Drafts/Components/UnderlineNav/UnderlineNav.Item',
+  component: UnderlineNavItem,
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <UnderlineNav aria-label="Repository">
+              <Story />
+            </UnderlineNav>
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    }
+  ],
```

Comment:
```suggestion
```
You should be able to delete this 😄 it exists in the root config for all stories already. If you add it here, it _might_ mess up the story source code

Reply:
oh 🙈 Thanks for letting me know about this!

Reply:
I deleted them all in one commit - fyi